app/data_providers/slate_providers/new_tab_slate_provider.py

```diff
@@ -52,8 +71,7 @@ def _get_tile_id_key(self, item: CorpusItemModel) -> str:
         """
         :return: A string that identifiers the scheduled surface, scheduled date, and CorpusItem.
         """
-        # TODO: When scheduledSurface.items is queried from the Graph, fill in the scheduledDate below. [DIS-452]
-        return f'{self.recommendation_surface_id}/{item.id}/<TODO: pull in ScheduledSurfaceItem.scheduledDate>'
+        return f'{self.recommendation_surface_id}/{item.id}/{self.corpus_api_client.get_scheduled_date(item.id)}'
```

Comment:
It won't fail in that case, because f-strings can take anything that has a string representation, which is pretty much anything in Python.
```python
>>> f'surface/123/{None}'
'surface/123/None'
```

Reply:
True, but are we ok with having None there?

Reply:
Good question. Yes, I think that's an acceptable way to degrade in case of a failure.

Here's a possible scenario:
- A bug is introduced in the Curated Corpus API that returns `null` for `scheduledDate`. (This is never expected to happen normally, because we're fetching scheduled items.)
- Recommendation API starts logs an error to Sentry.
- Recommendation API continues to return recommendations. The `tileId` will be different because the date part is missing. - There probably will be no impact at all, but if there is any, it would be some low-impact reporting or Thompson sampling effect. If Thompson sampling would start getting stale data, an alert would go off for the [recommendation freshness SLO](https://getpocket.atlassian.net/wiki/spaces/PE/pages/2927099995/Fx+NewTab+for+New+Markets+Reliability+Doc).
- Data Products will see the Sentry error in #log-data-products-alerts and respond to it.